packages/react-core/src/components/MenuToggle/MenuToggle.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/MenuToggle/menu-toggle';
+import { css } from '@patternfly/react-styles';
+import CaretDownIcon from '@patternfly/react-icons/dist/js/icons/caret-down-icon';
+import { BadgeProps } from '../Badge';
+
+export interface MenuToggleProps
+  extends Omit<React.DetailedHTMLProps<React.ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement>, 'ref'> {
+  /** Content rendered inside the toggle */
+  children?: React.ReactNode;
+  /** Additional classes added to the toggle */
+  className?: string;
+  /** Flag indicating the toggle has expanded styling */
+  isExpanded?: boolean;
+  /** Flag indicating the toggle is disabled */
+  isDisabled?: boolean;
+  /** Flag indicating the toggle should have primary styling */
+  isPrimary?: boolean;
```

Comment:
I think whatever is applied last would win out on the css if both happened to be applied, but I don't think they're intended to be used together. Our Dropdown component has these as separate props too, but I'm not sure if that was just one being added later / to avoid a breaking change.

I can update these props into `variant` instead, though I'm curious if `variant` should itself be an optional prop or if it should it have a `default` value as well for basic use cases that don't need either plain/primary styling? @tlabaj

Reply:
I think the variant is the right way.  And yes, we can add the default state to the variant.